var/spack/repos/builtin/packages/geant4/package.py

```diff
@@ -78,6 +78,7 @@ class Geant4(CMakePackage):
     variant("x11", default=False, description="Optional X11 support")
     variant("motif", default=False, description="Optional motif support")
     variant("qt", default=False, description="Enable Qt support")
+    variant("hdf", default=False, description="Enable HDF5 support", when="@10.4:")
```

Comment:
Can you spell this as `hdf5` to be more consistent with the other (dozens of) Spack packages?

```suggestion
    variant("hdf5", default=False, description="Enable HDF5 support", when="@10.4:")
```